src/shared/components/UploadButton/UploadButton.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {MutableRefObject, useRef} from 'react';
+
+import {Fab} from '@mui/material';
+import FileUploadOutlinedIcon from '@mui/icons-material/FileUploadOutlined';
+
+import {IPhoto} from '../../interfaces';
+import {useAppDispatch} from '../../hooks/redux_hooks';
+import {addPhoto} from '../../store/photoSlice';
+import {uploadPhoto} from '../../helpers/helpersUploadPhoto';
+
+import * as styles from './styles';
+
+const UploadButton = ({nameBtn}: {nameBtn: string}): JSX.Element => {
+  const dispatch = useAppDispatch();
+  const filePicker = useRef() as MutableRefObject<HTMLInputElement>;
+
+  const handlerPick = () => {
+    filePicker.current.click();
+  };
+
+  const handlerChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const files = event.target.files;
+    if (files) {
+      const callBack = (data: IPhoto) => {
+        dispatch(addPhoto(data));
+      };
+
+      for(const file of Array.from(files)){
+        uploadPhoto(file, callBack);
+      }
+    }
+  };
+
+  return (
+    <>
+      <input
+        ref={filePicker}
+        style={styles.InputHidden}
+        onChange={handlerChange}
+        multiple
+        type="file"
+        accept=".png,.webp,.jpg,.jpeg"/>
+      <Fab
+        sx={{
+          position: 'fixed',
+          top: 638,
```

Comment:
It must be at the bottom of the screen

Reply:
<img width="841" alt="Screenshot 2023-01-19 at 15 21 07" src="https://user-images.githubusercontent.com/110606859/213453339-70caa7cd-ee1e-42ae-86e5-34aefd606083.png">
This is what I see